### ISSUE TITLE
Win10Pro 10.0.18362.836 all green, but not working

### ISSUE THREAD

**Issue by Gh0sTG0** [stascorp/rdpwrap#1118]
Hi. This morning my pc with win10pro updated and wrapper stops working.
As always, I go to https://github.com/asmtron/rdpwrap/blob/master/res/rdpwrap.ini - Page with ini, downloaded it and replaced that ini in folder.
After I restart pc config starts saying that all is OK (just standart screen with all things green).
But, when I'm trying to connect, it allows only one user to work, another one is kicked from work. Like Wrapper is not working.
Then I tried this thing - https://github.com/stascorp/rdpwrap/issues/1083 . No changes.
What can I do to it start working, so both users can work at the same time?

**Comment by robbak** [comment]
Just to confirm, both users have separate user accounts on the system? If
you log in remotely using the same user account as the person logged in to
the local console, it will always detach the session from the console and
connect it remotely. It is best to give different users separate user
accounts on the local system. 'Multiple sessions per user' allows 2 people
to connect remotely using the same user account, and (maybe) allows a
person to get a new session at the console without disconnecting a remote
user.

**Comment by Gh0sTG0** [comment]
> Just to confirm, both users have separate user accounts on the system? If you log in remotely using the same user account as the person logged in to the local console, it will always detach the session from the console and connect it remotely. It is best to give different users separate user accounts on the local system. 'Multiple sessions per user' allows 2 people to connect remotely using the same user account, and (maybe) allows a person to get a new session at the console without disconnecting a remote user.

Ofc we are separate users. It's not new setup. It worked for ~2 years and already had some number of updates that needed to change ini. It worked yesterday, but this morning windows updated =(
PS tried to uninstall-reboot-install-reboot-(for emergency)replace ini-reboot. It not works.
PPS tried to allow multisession on one account. It didn't help to log in us into our accounts.

**Comment by Gh0sTG0** [comment]
So... Some strange things and now I don't have that problem more.
I just checked windows update and there was some windows 10 update for x64... smth like that (didn't screenshoot, sry) and one 07.20 for 09.19 update.
So, what I did:
-uninstall rdp.
-update win for that updates
-reload
-use https://github.com/asmtron/rdpwrap/blob/master/binary-download.md this instruction. No original installer, only launching autoupdate.bat
...
Profit. It works for me now.
